macros/conversations.cfg

```diff
@@ -1,7 +1,7 @@
 #textdomain wesnoth-A_New_Order
 #define AFTERBARNONSTORY
     [part]
-        # Konrad2 suggested replacing "was ended" with "was over" but I think "had ended" works better:
+        #po: Konrad2 suggested replacing "was ended" with "was over" but I think "had ended" works better:
```

Comment:
This line is fine, you don't need the po. Entire comment can go.

Reply:
> This line is fine, you don't need the po. Entire comment can go.

ok, removed in [f302ec2](https://github.com/nemaara/A_New_Order/pull/69/commits/f302ec2567e9da46ff40e29fe91592bee373ba27)